.prettierrc

```diff
@@ -1,5 +1,14 @@
 {
   "semi": false,
   "trailingComma": "es5",
-  "singleQuote": true
-}
+  "singleQuote": true,
+  "parser": "babel-flow",
+  "overrides": [
+    {
+      "files": "*.ts",
+      "options": {
+        "parser": "typescript"
```

Comment:
Here we use the `override` feature to tell prettier to use `typescript` parser for the ts files, otherwise the `babel-flow` parser.